src/hardware/mixer.cpp

```diff
@@ -1020,124 +1099,224 @@ class MIXER final : public Program {
-			             channel->volmain[1], channel->GetSampleRate(),
-			             channel->DescribeLineout().c_str());
-		lock.unlock();
+		if (showStatus)
+			ShowMixerStatus();
 	}
 
 private:
-	void ShowSettings(const char *name,
-	                  const float vol0,
-	                  const float vol1,
-	                  const int rate,
-	                  const char *lineout_mode)
-	{
-		WriteOut("%-11s %4.0f:%-4.0f  %+6.2f:%-+6.2f %8d   %s\n", name,
-		         static_cast<double>(vol0 * 100),
-		         static_cast<double>(vol1 * 100),
-		         static_cast<double>(20 * log(vol0) / log(10.0f)),
-		         static_cast<double>(20 * log(vol1) / log(10.0f)), rate,
-		         lineout_mode);
-	}
-
-	void ListMidi() { MIDI_ListAll(this); }
-
-	void AddMessages() 
+	void AddMessages()
 	{
 		MSG_Add("SHELL_CMD_MIXER_HELP_LONG",
 		        "Displays or changes the sound mixer settings.\n"
 		        "\n"
 		        "Usage:\n"
-		        "  [color=green]mixer[reset] [color=cyan]CHANNEL[reset] [color=white]VOLUME[reset] [/noshow]\n"
-		        "  [color=green]mixer[reset] [color=cyan]CHANNEL[reset] [color=white]LINEOUT[reset] [/noshow]\n"
+		        "  [color=green]mixer[reset] [color=cyan][CHANNEL][reset] [color=white]COMMANDS[reset] [/noshow]\n"
 		        "  [color=green]mixer[reset] [/listmidi]\n"
 		        "\n"
 		        "Where:\n"
-		        "  [color=cyan]CHANNEL[reset] is the sound channel to change the settings of.\n"
-		        "  [color=white]VOLUME[reset]  is either a percentage value between 0.0 and 100.0,\n"
-		        "          or a decibel value prefixed with [color=white]d[reset] (e.g. d-7.5).\n"
-		        "  [color=white]LINEOUT[reset] sets the channel's line out mode to one of the following:\n"
-		        "          [color=white]stereo, reverse, leftmono, rightmono[reset]"
+		        "  [color=cyan]CHANNEL[reset]  is the sound channel to change the settings of.\n"
+		        "  [color=white]COMMANDS[reset] is one or more of the following commands:\n"
+		        "    Volume:    [color=white]0[reset] to [color=white]100[reset], or decibel value prefixed with [color=white]d[reset] (e.g. [color=white]d-7.5[reset])\n"
+		        "               use [color=white]L:R[reset] to set the left and right side separately (e.g. [color=white]10:20[reset])\n"
+		        "    Lineout:   [color=white]stereo[reset], [color=white]reverse[reset] (for stereo channels only)\n"
+		        "    Crossfeed: [color=white]x0[reset] to [color=white]x100[reset]\n"
 		        "\n"
 		        "Notes:\n"
 		        "  Running [color=green]mixer[reset] without an argument shows the current mixer settings.\n"
-		        "  You can view the list of available MIDI devices with /listmidi.\n"
 		        "  You may change the settings of more than one channel in a single command.\n"
+		        "  If channel is unspecified, you can set crossfeed, reverb or chorus globally.\n"
+		        "  You can view the list of available MIDI devices with /listmidi.\n"
```

Comment:
I like all the wording improvements and use of color. 
Pleasure using the updated `mixer`!

Reply:
Great to hear some is appreciating it! 😆 I've certainly put a lot of thought into it and it went through several iterations; I think it was worth it in the end.

Reply:
It's feeling extremely polished; like a very fully-featured DOS audio control panel!